### PR TITLE
[CU-860rb0bda] Game-settings:  Add logic for all weapons

### PR DESCRIPTION
**Pull request by FlavianEng**
>__Note__
> This PR updates app fixtures. After merge, run `npm run fixture` and choose the option `both` or `app`.
> This PR is based on #56 

## Changelog: 
- Add tests for all weapons
- Create random shoot function for the random weapon
- Update app fixtures

## Checklist before requesting a review
-   [x] I have performed a self-review of my code
-   [x] My pull request title respect the naming convention: [**_ClickUp Id_**] **_summary_**

### CODE REVIEW

**Comment by FlavianEng** [comment]
Task linked: [CU-860rb0bda Game settings : add logic for all weapons](https://app.clickup.com/t/860rb0bda)

**Comment by FlavianEng** [comment]
Task linked: [CU-3c6jpeb Game settings : generate board game](https://app.clickup.com/t/3c6jpeb)

File: src/datasets/game-instance.dataset.ts [hunk 1]
```diff
@@ -322,13 +394,51 @@ const gameWeaponTriple: () => WeaponType = () => {
...
+    damageArea: [
+      [1, 0],
+      [1, -1],
+      [1, 1],
+      [0, -1],
+      [0, 0],
+      [0, 1],
+      [-1, 0],
+      [-1, -1],
+      [-1, 1],
```
Comment (MASBERNATQ) [inline src/datasets/game-instance.dataset.ts:424]: On va être sur ce type de dégat :
image

> Il te manque les 4 cases sur les extrémités

Comment (FlavianEng) [reply src/datasets/game-instance.dataset.ts:424]: J'etais pas du tout la desus

Comment (FlavianEng) [reply src/datasets/game-instance.dataset.ts:424]: Je corrige, c'est l'affaire de 20 minutes

Comment (FlavianEng) [reply src/datasets/game-instance.dataset.ts]: Done

**Review by MASBERNATQ — APPROVED** [review]

**Pull request merged**